src/Tribe/Aggregator/Record/Eventbrite.php

```diff
@@ -157,22 +157,26 @@ public static function filter_add_site_get_import_data( $args, $record ) {
 	 * @return array
 	 */
 	public static function filter_setup_do_not_override_post_status( $post_status, $event, $record ) {
-
-		if ( ! isset( $event['eventbrite']->status ) || ! isset( $record->meta['post_status'] ) ) {
-			return $post_status;
-		}
-
-		// We're only interested in the "(do not override)" post status.
-		if ( 'do_not_override' !== $record->meta['post_status'] ) {
-			return $post_status;
+                // override status if set within import
+                $status = isset( $record->meta['post_status'] ) ? $record->meta['post_status'] : $post_status;
+                if ( 'do_not_override' === $status ) {
+                    if( isset($event['eventbrite']->status) ){
+                        // draft if set, otherwise publish
+                        if( 'draft' === $event['eventbrite']->status ){
+                            $status = 'draft';
+                        }else{
+                            $status = 'publish';
+                        }
+                    }else{
+                        // if not set, assume publish
+                        $status = 'publish';
+                    }
+                    // If not draft, looked if listed. If not, set to private
+                    if( 'draft' !== $status && isset($event['eventbrite']->listed) && !tribe_is_truthy($event['eventbrite']->listed) ){
```

Comment:
* :no_entry_sign: Tabs must be used to indent lines; spaces are not allowed
* :no_entry_sign: Line indented incorrectly; expected 3 tabs, found 5
* :no_entry_sign: Space after opening control structure is required
* :no_entry_sign: No space before opening parenthesis is prohibited
* :no_entry_sign: Expected 1 space(s) after IF keyword; 0 found
* :no_entry_sign: There must be a space between if and the first parenthesis '('
* :no_entry_sign: Expected 1 spaces after opening parenthesis; 0 found
* :no_entry_sign: Expected 1 spaces before closing parenthesis; 0 found
* :no_entry_sign: There must be a single space after the first parenthesis
* :no_entry_sign: There must be a single space before the last parenthesis
* :no_entry_sign: Expected 1 space after "!"; 0 found
* :no_entry_sign: ! must be followed by a single space
* :no_entry_sign: Expected 1 spaces after opening parenthesis; 0 found
* :no_entry_sign: Expected 1 spaces before closing parenthesis; 0 found
* :no_entry_sign: There must be a single space after the first parenthesis
* :no_entry_sign: There must be a single space before the last parenthesis
* :no_entry_sign: Expected 1 space(s) after closing parenthesis; found 0
* :no_entry_sign: You must have a space after a closing parenthesis
* :no_entry_sign: Space between opening control structure and closing parenthesis is required

<sub>hash: 6e614c39354b37364deb70e791301031</sub>